src/Payments.sol

```diff
@@ -789,6 +794,15 @@ contract Payments is Initializable, UUPSUpgradeable, OwnableUpgradeable, Reentra
         return settleRailInternal(railId, maxSettleEpoch, true);
     }
 
+    function burn(uint256 _amount) internal {
+        require(msg.value >= _amount, "not enough native token to burn");
+        // f099 burn address
+        (bool success,) = address(0xff00000000000000000000000000000000000063).call{value: _amount}("");
```

Comment:
I need to check if this will work

Reply:
@Kubuxu This is how we burn in the `PDPVerifier`:

https://github.com/FilOzone/pdp/blob/d051560b953072942ea5fc8ae8d31b9b0c2e7021/src/PDPVerifier.sol#L30